distrobox-host-exec

```diff
@@ -117,7 +117,7 @@ if [ "${verbose}" -ne 0 ]; then
 fi
 
 # Check we're running inside a container and not on the host
-if [ -z "${CONTAINER_ID:-}" ]; then
```

Comment:
CONTAINER_ID is docker-specific it seems so wouldn't work in podman, although we could add a `&& [ -z "${CONTAINER_ID:-}" ]` to the overall check if we wanted

Reply:
I think we can just check the 2 files and CONTAINER_ID

CONTAINER_ID is not a docker-specific thing, but a distrobox specific thing that we add in enter here:
https://github.com/89luca89/distrobox/blob/065a0965076b1bfcea7b952c255d05f041c9a7a3/distrobox-enter#L318

We could move this `--env` from enter to create, and we should be able to detect this :+1:

Reply:
Neither of these three things can be guaranteed in podman, that's why I do these three and the $container env var

Reply:
Both here (host-exec) and in export, the CONTAINER_ID is guaranteed by the `distrobox-enter` that injects this variable in the env

We must pass trough an enter so it's safe to assume that CONTAINER_ID is present